package.json

```diff
@@ -7,9 +7,11 @@
     "@material-ui/core": "4.11.4",
     "@material-ui/icons": "4.11.2",
     "@reduxjs/toolkit": "1.6.0",
+    "chart.js": "^3.5.0",
```

Comment:
Let's remove the `^` char here and run `yarn install` again.

Reply:
done